### ISSUE TITLE
How best to fork and edit the backend container logic

### ISSUE THREAD

**Issue by cm2435** [fauxpilot/fauxpilot#140]
Firstly, really cool project, was a great primer into fastertransformer which I hadn't used as a runtime previously, really nicely implemented!

I'm looking into using some LLM's for search tasks, as part of my work I would like to be able to generate a log prob tensor for each generation of shape 
```
[tokenizer_vocab_size, number_tokens_generated]
```
rather than a flat list of the logprobs of only the generated tokens. Essentially, I want for each token choice sampled in the .generate() method I would like a tensor of the log probability of all of the tokens in the vocabulary to implement some neural search re ranking.
I can do this by overriding the model.generate method fine note. 

When trying to integrate into this project for my use-case on a forked branch, I noticed that the docker compose builds from a container on dockerhub built by @moyix. I'm not hugely clear how to rerun with edits to 'python_backend/models.py' or the proxy in such a way it will actually build a changed container when running launch.sh.

TLDR- how I edit the model inference files in such a way that it actually builds a new docker compose? 

Thanks so much in advance!.

**Comment by thakkarparth007** [comment]
From what I understand, you want to build the docker container that hosts the actual model (for python backend), right? If you make changes to model.py, you just need to run `docker compose build` from the root directory. If you want it to get built automatically in launch.sh, you can modify the last line in it to be `docker-compose up --build ...` (pass the `--build` flag). Ideally...we should do this ourselves.

**Comment by thakkarparth007** [comment]
Hey @cm2435, let me know if this PR fixes things for you.
